docker-compose.yml

```diff
@@ -48,16 +48,30 @@ services:
       - /etc/localtime:/etc/localtime
       - /var/log/multi-cloud:/var/log/multi-cloud
 
+  redis:
+    image: redis
+    container_name: redis
+    command: redis-server
+    ports:
+      - "6379:6379"
+
   s3:
     image: opensdsio/multi-cloud-s3
     environment:
       - MICRO_REGISTRY=mdns
-      - DB_HOST=datastore:27017

```

Comment:
is the mongo image and container section removed from the compose file ?

Reply:
@rhsakarpos thanks for your review. according to the discussion before, multi-cloud s3 will use tidb, so, in docker-compose.yml, we use tidb for testing.